packages/node-opentelemetry/src/otel.ts

```diff
@@ -134,17 +140,40 @@ export const initSDK = (config: SDKConfig) => {
     );
   }
 
+  hdx(
+    stopOnTerminationSignals
+      ? 'stopOnTerminationSignals enabled'
+      : 'stopOnTerminationSignals disabled (user is responsible for graceful shutdown on termination signals)',
+  );
+
+  function handleTerminationSignal(signal: string) {
+    hdx(`${signal} received, shutting down...`);
+    _shutdown().finally(() => process.exit());
+  }
+
   // Graceful shutdown
-  process.on('SIGTERM', () => {
-    hdx('SIGTERM received, shutting down');
-    hdxConsoleInstrumentation.disable();
-    hyperDXGlobalContext.shutdown();
-    sdk
-      .shutdown()
-      .then(
-        () => console.log(`${LOG_PREFIX} otel SDK shut down successfully`),
-        (err) => console.log(`${LOG_PREFIX} Error shutting down otel SDK`, err),
-      )
-      .finally(() => process.exit(0));
-  });
+  if (stopOnTerminationSignals) {
+    process.on('SIGTERM', () => {
+      handleTerminationSignal('SIGTERM');
+    });
+    process.on('SIGINT', () => {
```

Comment:
Adding SIGINT as well, so ctrl+c will be captured too (not that it should matter too much in prod)